lib/graphiti/adapters/active_record.rb

```diff
@@ -304,7 +304,7 @@ def destroy(model_instance)
       end
 
       def close
-        ::ActiveRecord::Base.clear_active_connections!
+        ::ActiveRecord::Base.connection_handler.clear_active_connections!
```

Comment:
if ActiveRecord.version > '7.2'
  ::ActiveRecord::Base.connection_handler.clear_active_connections!
else
  ::ActiveRecord::Base.clear_active_connections!
end

Reply:
Let's assume that not everyone would be able to upgrade to 7.2+ as fast as we need to do it ;)

Reply:
as a quick fix, in the meantime, we could  use something like the following:
`ActiveRecord::Base.class.delegate :clear_active_connections!, to: :connection_handler`
somewhere in an initializer, or such.

Reply:
or a much cleaner version:
```
ActiveRecord::Base.define_singleton_method :clear_active_connections! do
  connection_handler.clear_active_connections!
end
```